### ISSUE TITLE
ElementVisibilityChanged event

### ISSUE THREAD

**Issue by anbare** [Microsoft/AdaptiveCards#1638]
## The problem

When cards are in a scrolling list, like instant messages, and the user clicks on a ShowCard action, the host needs to know that they should scroll down to show the new card content that appears. Therefore, hosts need an event they can listen to when element visibility has changed, so they know which content they should ensure is in the scroll view.

## Implementation status

- [ ] TypeScript
- [ ] UWP
- [ ] .NET (#842)
- [ ] iOS
- [ ] Android
- [ ] Documentation

**Comment by tnev** [comment]
@andrewleader When this is finished, will we be able to get a callback for the "ToggleVisibility" event on iOS?

**Comment by andrewleader** [comment]
That's correct, please let us know your scenario and why you need that feature so we can help prioritize!

**Comment by tnev** [comment]
@andrewleader thanks for the reply! We need this so we can trigger a height update for the table view we keep the cards in on iOS. This is similar to what we do when we receive _didChangeVisibility_ or _didChangeViewLayout_ events.
